tests/Unit/inc/ThirdParty/Hostings/HostResolver/getHostService.php

```diff
@@ -34,10 +34,6 @@ public function testShouldReturnExpectedValue( $expected ) {
 			case 'spinupwp':
 				putenv( 'SPINUPWP_CACHE_PATH=/wp-content/spinupwp-cache/' );
 				break;
-			case 'wpengine':
-				require_once WP_ROCKET_TESTS_FIXTURES_DIR . '/inc/ThirdParty/Hostings/WPEngine/wpe_param.php';
-				require_once WP_ROCKET_TESTS_FIXTURES_DIR . '/inc/ThirdParty/Hostings/WPEngine/WpeCommon.php';
-				break;
```

Comment:
Why do we remove those?

Reply:
For unit tests, those files are useless, you will find that no test fails after removal.